quantum/mcu_selection.mk

```
@@ -54,7 +53,7 @@ ifneq (,$(filter $(MCU),atmega32u4 at90usb1286))
   # LUFA specific
   #
   # Target architecture (see library "Board Types" documentation).
-  ARCH ?= AVR8
+  ARCH = AVR8
```

I think it may be a good idea to leave this as soft set.  Though, not sure about that. 

```suggestion
  ARCH ?= AVR8
```

---

I don't know what else it could possibly be set to - the 16/32U4 and USB64/128 are intrinsically AVR8. :/